manager/assets/modext/widgets/resource/modx.panel.resource.js

```diff
@@ -752,16 +752,33 @@ Ext.extend(MODx.panel.Resource,MODx.FormPanel,{
             ,getState: function() {
                 return { collapsed: this.collapsed };
             }
-            ,items: [{
-                xtype: 'xcheckbox'
-                ,boxLabel: _('resource_published')
-                ,hideLabel: true
-                ,description: '<b>[[*published]]</b><br />'+_('resource_published_help')
-                ,name: 'published'
-                ,id: 'modx-resource-published'
-                ,inputValue: 1
-                ,checked: parseInt(config.record.published)
-            },{
+            ,items: [
+                [{
+                layout: 'column'
+                ,border: false
+                ,padding: '10px 0 0'
```

Comment:
```suggestion
```

Reply:
@JoshuaLuckers Joshua, looks like if I do like you proposed then the layout will be changed from the table view to the simple lines. Please see the attached files below (before and after your changes):
![before](https://user-images.githubusercontent.com/6464115/55947868-fbec7a80-5c57-11e9-9405-d5931c5063d0.png)
![after](https://user-images.githubusercontent.com/6464115/55947866-fb53e400-5c57-11e9-9aa3-71aecb52fc49.png)

I believe it would be better to show those options in one line. However I'm not sure how to make them both to take 50% of the available width... Do you have any idea?

Thanks,
Igor

Reply:
My screen was smaller and they didn't align properly, that's why I moved them to their own row instead. The length of the text in the label is also depending on the lexicon and might break the alignment (can be longer/shorter in other languages).

Reply:
In my opinion, the variant with a width of 50% is better.

Reply:
I agree with it looking better (when there is enough space). We have to figure out if there is a better solution where it looks good in all (or most) situations.

Reply:
I checked languages, everywhere there is enough space for 50%. The only thing that with small windows need to move the deleted line to a new line without padding.

![delete_right](https://user-images.githubusercontent.com/12523676/55959976-f0f81100-5c7c-11e9-97bc-59dadb020395.png)

Reply:
Guys, I've tried a lot of different formats here including, for example, table view:

`...
                layout: 'table'
                ,layoutConfig: {
                    columns: 2
                }
...
`
However probably due to limitation of the current design (it's not responsive) the result looks quite ugly :(

Table View:
![table-view](https://user-images.githubusercontent.com/6464115/55984234-b48cdb00-5ca5-11e9-87da-9445680a7d0b.png)

So currently I'm thinking that the Joshua's idea to distribute the elements by each own row can be the best one. Any other ideas?

Reply:
Maybe there is a class that we can use in combination with the `column` layout that has the padding and automatically puts the label on a new row when needed. I'm sure we can figure this out 😎.

Reply:
Yeah, I would be happy to know if there is such class and how it is named :) Maybe someone else will join our discussion and let us know then.

Reply:
I tried various things without luck. ExtJS 3 really isn't mobile friendly 😞.

Reply:
So, what would you recommend, Joshua? Should I change the code or leave it as is then?

Reply:
@JoshuaLuckers, what's your recommendation on this?

Reply:
I recommend applying my suggestions and put them in a `row` layout instead of `column`. It might not look perfect but at least it doesn't break alignment (which looks worse in my opinion).